app/gui/src/lib.rs

```diff
@@ -40,6 +40,7 @@
 #![feature(option_result_contains)]
 #![feature(trait_alias)]
 #![feature(result_into_ok_or_err)]
+#![feature(result_option_inspect)]
```

Comment:
(Note: added to allow use of the [Result::inspect_err](https://doc.rust-lang.org/std/result/enum.Result.html#method.inspect_err) method.)